code/starfield/starfield_flags.h

```diff
@@ -0,0 +1,15 @@
+#ifndef STARFIELD_FLAGS_H
+#define STARFIELD_FLAGS_H
+
+#include "globalincs/flagset.h"
+
+namespace Starfield {
+
+	FLAG_LIST(Background_Flags){
+		Corrected_angles_in_mission_file = 0,	// angles are saved to the mission file correctly; if this flag is not present they are saved using incorrect math
```

Comment:
The angles are *always* stored correctly in memory now.  This flag just contains whether correct angles are saved or incorrect angles are saved.

Reply:
Well, since saving corrected angles is now the default behavior, perhaps it'd be more logical to make the flag represent the exception, e.g., `Save_angles_using_legacy_format`?

Or, failing that, I suggest renaming this flag to something like `Save_angles_as_corrected`.

Or, better still, maybe use this flag only when FRED is running?

The problem with `_in_mission_file` is that it makes it sound like the angles in the mission file on disk are saved in corrected format, which may not necessarily be the case. For example, if we're running FSO and loading an old mission file that doesn't use corrected angles, then applying this flag doesn't make sense to me.

What do you think?

Reply:
Let me try to clarify.  The angles are always correct *in memory*, that is, in the internal game state, so that the correct angles are available for scripts and such.  But they are only saved in the correct format if the flag is enabled.  If the flag is not enabled, they are saved in the incorrect format.

When the mission is loaded, the angles are converted if the flag is not found, and left alone if the flag is found.

The flag must be set up so that it is active if and only if the background is saved to the mission file with correct angles.  This is because older missions with incorrect angles don't have the flag.